runtimes/spiritnet/src/lib.rs

```diff
@@ -18,6 +18,16 @@
 
 //! The KILT runtime. This can be compiled with `#[no_std]`, ready for Wasm.
 #![cfg_attr(not(feature = "std"), no_std)]
+#![warn(
+	clippy::integer_arithmetic,
+	clippy::integer_division,
+	clippy::as_conversions,
+	clippy::missing_panics_doc,
+	clippy::missing_errors_doc,
+	clippy::float_arithmetic,
+	clippy::cast_possible_wrap
+)]
+#![deny(clippy::index_refutable_slice, clippy::indexing_slicing)]
```

Comment:
We treat all warnings as errors anyway in the CI, so I don't think it's necessary to make the distinction here. Better to have them all in one place.